sunpy/io/cdf.py

```diff
@@ -170,6 +170,9 @@ def read_cdf(fname):
                 '1/(cm**2-s-sr-MeV/nuc.)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
                 '1/(cm^2 sec ster MeV)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
 
+                'particles / (s cm^2 sr MeV)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
+                'particles / (s cm^2 sr MeV/n)': 1 / (u.cm**2 * u.s * u.sr * u.MeV),
```

Comment:
Stupid question but the `/n` doesn't affect the final unit?

Reply:
> Stupid question but the `/n` doesn't affect the final unit?

No, it doesn't affect the units. "/n" stands for the "per nucleus", it's a division by a dimensionless integer. This is in this context always connected to the energy, and the unit is always that of energy. (It's used to distinguish between the energy _per particle_ or _per nucleus_ -  where e.g. an alpha particle is _one particle_ but consists of _four nuclei_.)

Reply:
it may be good just to add a comment after that here?

Reply:
Lines [168](https://github.com/sunpy/sunpy/blob/8c620fecf72b5020ae796c3284487beb5db3aac3/sunpy/io/cdf.py#L168) and [170](https://github.com/sunpy/sunpy/blob/8c620fecf72b5020ae796c3284487beb5db3aac3/sunpy/io/cdf.py#L170) already provide exactly the same physical quantity, only with different item names. Should we add comments to this everywhere? Doesn't seem necessary to me...

Reply:
yeah you're right - no common formats anywhere :P I don't think it needs a comment